### ISSUE TITLE
Controlling compression of wheels built in an unified way between the build backends

### ISSUE THREAD

**Issue by KOLANICH** [takluyver/flit#398]
https://github.com/pypa/build/issues/258

Unfortunately, each build backend has an own way to build wheels, so each one has to get an own issue. I wonder if the interface for specifying compression can be standardized.

**Comment by takluyver** [comment]
What's the use case this is meant to address?

If you want to make installation faster by avoiding compressing and decompressing the data, it may be better to return to an idea that was part of the PEP 517 discussions, of an interface to create an 'unpacked wheel' - i.e. a directory with the same contents as a wheel zip file. That would give both the frontend and the backend more possibilities for speeding things up (e.g. writing files in parallel, moving rather than copying).

**Comment by KOLANICH** [comment]
>If you want to make installation faster by avoiding compressing and decompressing the data, it may be better to return to an idea that was part of the PEP 517 discussions, of an interface to create an 'unpacked wheel' - i.e. a directory with the same contents as a wheel zip file.

Yes, and not only. Increasing compression can also be useful. Though unpacked wheel is an ultimate solution of all the questions related to wheel compression, since I can use 7z then to compress.
